### PR TITLE
added Niravprajapati1

### PR DESCRIPTION
**Pull request by Niravprajapati1**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1503"></a>

### CODE REVIEW

File: public/data/Niravprajapati1.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      "name": "Follow me on showwcase",
+      "url": "https://www.showwcase.com/nirav",
+      "icon": "link"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started learning java and dsa",
+      "date": "june 2022",
+      "icon": "global",
```
Comment (kumarsonsoff3) [inline public/data/Niravprajapati1.json:32]: You might wanted to add `globe` here!? 🛠

```suggestion
      "icon": "globe",
```

File: public/data/Niravprajapati1.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      "color": "green",
+      "description": "Started contributing to Open Source projects"
+    },
+    {
+      "title": "created first virtual machine",
+      "date": "May 2022",
+      "icon": "link",
+      "color": "blue",
+      "description": "learnt how to create virtual machine in Google cloud.",
+      "url": "https://coursera.org/verif y/TFQD5ZKM7MEZ"
```
Comment (kumarsonsoff3) [inline public/data/Niravprajapati1.json:49]: **There is a broken link!** Please remove the space from the link! 🛠

```suggestion
      "url": "https://coursera.org/verify/TFQD5ZKM7MEZ"
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Minor Fixes Required! 🛠

**Comment by Niravprajapati1** [comment]
I changed global to globe and updated broken link

**Review by Vyvy-vi — APPROVED** [review]
LGTM! Thanks for adding your profile to LinkFree!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by Niravprajapati1** [comment]
Thanks 🙌

**Review by kumarsonsoff3 — APPROVED** [review]
Thanks for the changes! LGTM 👍

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing!

**Pull request merged**